ETL/Configuration/ETLConfiguration.cs

```diff
@@ -7,34 +7,32 @@
 
```

Comment:
Why modify this file?

Reply:
I think it just automatically reformatted it